### ISSUE TITLE
Add: Display referencing NewGRF set when query is used on rail, road and tram tiles

### ISSUE THREAD

**Issue by kevinfields777** [OpenTTD/OpenTTD#8779]
Currently when you use the Land Area Information query tools on a tile with a NewGRF building or object on it, it will display which NewGRF the sprite comes from.  This should be extended when querying rail, road and tram tiles.

**Comment by perezdidac** [comment]
I am happy to take ownership of this task, I think I know where to make the changes, will keep this thread posted.

**Comment by perezdidac** [comment]
![image](https://user-images.githubusercontent.com/2025406/109548318-5cba1180-7a81-11eb-873d-7249f8f4d1e9.png)

Yeah I managed to do it, I will make sure this is maintainable and clean enough to be submitted and will send out a PR.

**Comment by 2TallTyler** [comment]
@perezdidac One case to consider, if you haven't already, is a road from one NewGRF and a tram track from another.

**Comment by perezdidac** [comment]
Yep I am totally considering it, changing a few things to be able to show the NewGRF for each track

**Comment by perezdidac** [comment]
Note for devs: plz assign this to myself, https://github.com/OpenTTD/OpenTTD/pull/8794 implements this.

**Comment by kevinfields777** [comment]
> Note for devs: plz assign this to myself, #8794 implements this.

Just curious if you had made any advancement since this was assigned to you?
